syne_tune/optimizer/schedulers/single_fidelity_scheduler.py

```diff
@@ -103,7 +106,11 @@ def on_trial_result(self, trial: Trial, result: Dict[str, Any]) -> str:
         metric = [
             result[metric_name] * self.metric_multiplier for metric_name in self.metrics
         ]
```

Comment:
Can we rename here and bellow with `metrics`? (given that we have a list, metric is a misnommer)
```suggestion
        metrics = [
            result[metric_name] * self.metric_multiplier for metric_name in self.metrics
        ]
```